src/App.test.js

```diff
@@ -1,24 +1,44 @@
 import React from "react";
-import { render } from "@testing-library/react";
-import { getAllByTestId } from "./utils/test-utils.js";
+import '@testing-library/jest-dom/extend-expect'
+import { render, fireEvent } from "@testing-library/react";
+import { getByTestId, getAllByTestId } from "./utils/test-utils.js";
 
 // component to be tested.
 import App from "./App.js";
+import Login from "../src/components/auth/login";
```

Comment:
Why was Login tested in this file?

Reply:
To test that the components dynamically render on App.js